### PR TITLE
Implement LDAP support

### PR DESCRIPTION
**Pull request by lindskogen**

### CODE REVIEW

**Comment by xPaw** [comment]
I hope you don't actually mean to remove password changing ability completely. You'd want to disable it if ldap support is enabled, but not remove it completely.

File: defaults/config.js [hunk 1]
```diff
@@ -269,5 +269,40 @@ module.exports = {
...
+		// @default  false
+		//
+		enable: false,
+
+		//
+		// LDAP server URL
+		//
+		// @type     string
+		//
+		//url: "ldaps://example.com",
```
Comment (xPaw) [inline defaults/config.js]: There's no need to comment out default values.

**Comment by lindskogen** [comment]
oops

**Comment by maxpoulin64** [comment]
I see you closed the pull request. If you still want to add this feature please consider reopening this one when you're ready to avoid spamming new PRs and splitting the discussion accross multiple threads.

To update a PR you just need to ammend or rebase your commits and force-push again to your branch: this will automatically update the PR as well ;)

**Comment by lindskogen** [comment]
I actually meant to merge this fix into our local fork of the project

**Pull request closed without merging**